package.json

```diff
@@ -32,28 +33,32 @@
   "license": "MIT",
   "scripts": {
     "build:node": "esbuild src/node.ts --outfile=lib/node.cjs --bundle --sourcemap --platform=node --external:cosmokit --external:supports-color --target=es2019",
-    "build:browser": "esbuild src/browser.ts --outfile=lib/browser.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --external:supports-color --target=esnext",
-    "build": "yarn build:node && yarn build:browser && yarn tsc -b",
+    "build:browser": "esbuild src/browser.ts --outfile=lib/browser.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --external:util --target=esnext",
```

Comment:
`--external:util` does not *fix* the problem but only bypasses it.

If you can't fix it you can leave it alone instead of concealing the problem.

Reply:
> `--external:util` does not _fix_ the problem but only bypasses it.
> 
> If you can't fix it you can leave it alone instead of concealing the problem.

Okey. I'm giving up on solving this problem.